src/Type/Enum/LinkToEnum.php

```diff
@@ -0,0 +1,7 @@
+<?php
+namespace WPGraphQL\Type;
+
+register_graphql_enum_type( 'LinkToEnum', [
+  'description' => __( 'Destination type of link', 'wp-graphql' ),
+  'values' => array( 'NONE', 'POST', 'FILE', 'CUSTOM' )
```

Comment:
I'm not seeing this Type used anywhere in this PR. . .what am I missing? 

Also, this seems like it could also be dynamically generated based on the Types of content registered to the WordPress install (maybe?) tried to look at where it's used for context but not seeing it.